WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -144,11 +144,11 @@ public LoadWalletViewModel(WalletManagerViewModel owner, LoadWalletType loadWall
 				IoHelpers.OpenBrowser(x);
 			});
 
-			OpenBrowserCommand.ThrownExceptions.Subscribe(Logger.LogWarning<LoadWalletViewModel>);
-			LoadCommand.ThrownExceptions.Subscribe(Logger.LogWarning<LoadWalletViewModel>);
-			TestPasswordCommand.ThrownExceptions.Subscribe(Logger.LogWarning<LoadWalletViewModel>);
-			OpenFolderCommand.ThrownExceptions.Subscribe(Logger.LogWarning<LoadWalletViewModel>);
-			ImportColdcardCommand.ThrownExceptions.Subscribe(Logger.LogWarning<LoadWalletViewModel>);
+			OpenBrowserCommand.ThrownExceptions.Subscribe(ex => Logger.LogWarning(ex));
```

Comment:
This comment is just to say that I like more how it was before. I mean, there is no need to create a lambda object here.

Reply:
It doesn't work that way anymore:(

![image](https://user-images.githubusercontent.com/9156103/64620229-c6ed7b00-d3e3-11e9-8bd9-76f10fcf425b.png)